frontend/src/_layouts/base.html

```diff
@@ -17,7 +17,7 @@
     {% render 'partials/gtm.html', tag-id: gtm-tag-id -%}
   </head>
 
-  <body class="flex min-h-screen flex-col overflow-y-scroll overlayed:overflow-hidden">
```

Comment:
This line was using the `data-state` plugin for `overlayed`. This is why the class sorter moved this class to the beginning of the line: all unknown classes go the beginning. And now `overlayed` is an unknown variation.

We need to replace this with something like `data-[overlayed]`. But besides this we need to actually trigger this data attribute from the backdrop. The `overflow-hidden` class is required to prevent body scrolling in the background while there is an overlay on top of it.

Reply:
`overlayed ` and `*-y-scroll` removed

Reply:
`data-[overlayed]` is added from backdrop component